resources/linux/archlinux/PKGBUILD.in

```diff
@@ -0,0 +1,16 @@
+pkgname=<%= appFileName %>
+pkgver=<%= version %>
+pkgrel=1
+pkgdesc="Chrome-based text editor from Github"
+arch=('<%= arch %>')
+url="http://atom.io"
+license=('MIT')
+provides=('atom')
+conflicts=('atom-editor' 'atom-editor-bin')
+depends=('nss' 'gtk2' 'libnotify' 'alsa-lib' 'gconf' 'libxtst' 'nodejs' 'libgnome-keyring' 'python' 'desktop-file-utils')
+makedepends=('npm')
+md5sums=()
```

Comment:
I wouldn't list `nodejs` or `npm` as a dependency. As far as I know it isn't as atom packages it's own copy of node and npm. 

Actually.. we should just use the depends from `atom-editor-bin`:

```
depends=('git' 'gconf' 'gtk2' 'libnotify' 'libxtst' 'nss' 'python2' 'xdg-utils' 'desktop-file-utils' 'alsa-lib' 'libgnome-keyring')
optdepends=('gvfs')
```

And you don't need `npm` as a `makedepends` as atom is already built. We just need `cp` here.